registry/donor/utils.py

```diff
@@ -26,13 +26,17 @@ def repair_two_semicolons(line):
     return repaired_line
 
 
-def repair_line_part_by_part(line):
+def repair_line_part_by_part(line):  # noqa: C901 FIXME
```

Comment:
To čo je?

Reply:
Příliš komplexní funkce vyžadující refraktoring.